### ISSUE TITLE
A new Feature for Fullscreen Themes

### ISSUE THREAD

**Issue by MoeinLM** [Lacro59/playnite-successstory-plugin#562]
I think it would be perfect if we had a list or something in Fullscreen Themes that show's how many trophies we have in total and how much progress we have in different games.

that way we don't have to change into desktop mode every time.

**Comment by saVantCZ** [comment]
To add details, data are already here for desktop mode, but we need exposed bindings for total count of all games (best with separated rarity) so fullscreen theme creators can create summary of all games (really useful for example on my PS5 theme)

Also, if possible, on desktop theme its possible to have visible achievements in game list for each game alone, its not possible to do in fullscreen because it will always show data only for currently selected game, making all strings same (for example if we add string for unlocked count into cover image of game, it will show in all games same data of currently selected game)

Sending love from all theme creators ❤️

**Comment by MoeinLM** [comment]
Yeah this feature would especially look good on PS5reborn theme (which I'm currently using as my main theme)

And by the way thanks a lot for updating your theme so often, can't wait to see this feature on it.

**Comment by saVantCZ** [comment]
Bumping this

Please consider adding exposed Total achievements data for fullscreen theme usage ;)
